patient-dao/src/main/java/hu/kuncystem/patient/dao/user/JDBCUserGroupDao.java

```diff
@@ -66,31 +68,56 @@ public UserGroup mapRow(ResultSet rs, int row) throws SQLException {
     private JdbcOperations jdbc;
 
     public List<User> getAllUserFromGroup(UserGroup group) {
-        return jdbc.query(SQL_GET_USERS_FROM_GROUP, new JDBCUserDao.UserRowMapper(), group.getId());
+        try {
+            return jdbc.query(SQL_GET_USERS_FROM_GROUP, new JDBCUserDao.UserRowMapper(), group.getId());
+        } catch (EmptyResultDataAccessException e) {
+            return null;
+        } catch (DataAccessException e) {
+            e.printStackTrace();

```

Comment:
This is a general comment on your code:
I suggest that the DAO layer throws DataAccessException (or a custom one created by you; could be RuntimeException) and let the service layer or the GUI controller do something about it.
Simply catching and logging it would not indicate the error to the end user and this is not what a real-world customer wants